manifests/rbac.yaml

```diff
@@ -35,4 +35,42 @@ roleRef:
 subjects:
   - kind: ServiceAccount
     name: harvester-node-manager
-    namespace: harvester-system
\ No newline at end of file
+    namespace: harvester-system
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: harvester-node-manager-webhook
+rules:
+  - apiGroups: [ "node.harvesterhci.io" ]
+    resources: [ "*" ]
+    verbs: [ "get", "watch", "list" ]
+  - apiGroups: [ "" ]
+    resources: [ "secrets", "configmaps" ]
```

Comment:
@connorkuehl Before merging the code, does the webhook need access to secrets here? Thanks!

Reply:
@bk201 I believe so, because it will write its ca and cert as secrets.

Reply:
Thanks!